x-pack/plugins/beats_management/public/components/config_list.tsx

```diff
@@ -35,22 +40,43 @@ export const ConfigList: React.SFC<ComponentProps> = props => (
       },
       {
         field: 'module',
-        name: 'Module',
+        name: props.intl.formatMessage({
+          id: 'xpack.beatsManagement.tagConfigTable.moduleColumnName',
+          defaultMessage: 'Module',
+        }),
         truncateText: false,
         render: (value: string) => {
-          return value || 'N/A';
+          return (
+            value ||
+            props.intl.formatMessage({
+              id: 'xpack.beatsManagement.tagConfigTable.moduleColumn.notAvailibaleLabel',
+              defaultMessage: 'N/A',
+            })
+          );
         },
       },
       {
         field: 'description',
-        name: 'Description',
+        name: props.intl.formatMessage({
+          id: 'xpack.beatsManagement.tagConfigTable.descriptionColumnName',
+          defaultMessage: 'Description',
+        }),
       },
       {
-        name: 'Actions',
+        name: props.intl.formatMessage({
+          id: 'xpack.beatsManagement.tagConfigTable.actionsColumnName',
+          defaultMessage: 'Actions',
+        }),
         actions: [
           {
-            name: 'Remove',
-            description: 'Remove this config from tag',
+            name: props.intl.formatMessage({
+              id: 'xpack.beatsManagement.tagConfigTable.actions.removeButtonAriaLabel',
```

Comment:
I didn't update the last segment `.removeButtonAriaLabel` because this label is displayed as `aria-label` attribute's value:

![image](https://user-images.githubusercontent.com/11524637/48769489-d7096200-eccc-11e8-9a63-c67b2acbc4f1.png)